docs/api.md

```diff
@@ -4442,13 +4442,17 @@ This method returns boxes of the element, or `null` if the element is not visibl
   - `button` <"left"|"right"|"middle"> Defaults to `left`.
   - `clickCount` <[number]> defaults to 1. See [UIEvent.detail].
   - `delay` <[number]> Time to wait between `mousedown` and `mouseup` in milliseconds. Defaults to 0.
+  - `offset` <[Offset]> Offset in pixels relative to the border box of the element.
```

Comment:
I think by overwriting this name in `check_public_api`, it now expects this to be of type `Object`

Reply:
it seems to complain nevertheless. Let's check on CI.